uPortal-utils/uPortal-utils-core/build.gradle

```diff
@@ -78,6 +80,9 @@ dependencies {
     compileOnly "${servletApiDependency}"
 }
 
+sourceCompatibility = '9'
```

Comment:
:thinking: its interesting that this needed to be set here.
I thought `allprojects` would have covered this :man_shrugging: